### PR TITLE
Fix manticore buddy

### PR DESCRIPTION
**Pull request by djklim87**

### CODE REVIEW

**Comment by djklim87** [comment]
Waiting for https://github.com/manticoresoftware/docker/pull/27

**Comment by sanikolaev** [comment]
> Waiting for [manticoresoftware/docker#27](https://github.com/manticoresoftware/docker/pull/27)

Approved

File: charts/manticoresearch/values.yaml [hunk 1]
```diff
@@ -29,7 +30,7 @@ balancer:
       port: 9312
       targetPort: 9312
   config:
-    path: /etc/manticoresearch/configmap.conf
+    path: /mnt/configmap.conf
```
Comment (sanikolaev) [inline charts/manticoresearch/values.yaml:33]: What's the reason for this change?

Comment (sanikolaev) [reply charts/manticoresearch/values.yaml:33]: Otherwise there's a conflict in the entrypoint script.
